devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -626,6 +626,9 @@ private def handleBatteryAlarmReport(cmd) {
 	def deviceName = device.displayName
 	def map = null
 	switch(cmd.zwaveAlarmEvent) {
+		case 0x01: //power has been applied, check if the battery level updated
```

Comment:
Does this work with local execution?

Reply:
Yeah. LE only handles types 1, 7, 2, and 5. Everything else falls through. @kianooshST can you  confirm?